db/migrate/20210319153015_create_versions.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# This migration creates the `versions` table, the only schema PT requires.
+# All other migrations PT provides are optional.
+class CreateVersions < ActiveRecord::Migration[6.1]
+  # The largest text column available in all supported RDBMS is
+  # 1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
+  # so that MySQL will use `longtext` instead of `text`.  Otherwise,
+  # when serializing very large objects, `text` might not be big enough.
```

Comment:
Ditto, I wonder if we need this in postgres.

I also wonder how large we expect our objects to get - I would be shocked if they were longer than 10k characters.

Reply:
I've removed it and moved to json rather than text.